axis.go

```diff
@@ -355,12 +350,21 @@ type DefaultTicks struct{}
 
 var _ Ticker = DefaultTicks{}
 
+var displayPrecision int
```

Comment:
This cannot be used as a global. If I have more than one plot being generated at the same time by different goroutines, now we have a race.

Reply:
Changed that. I hope now it will not cause race.